### PR TITLE
add seed data for status

### PR DESCRIPTION
**Pull request by SimonLab**
ref: https://github.com/dwyl/app/issues/247#issuecomment-561708026
Create status data

### CODE REVIEW

File: priv/repo/seeds.exs [hunk 1]
```diff
@@ -1,11 +1,2 @@
...
-#
-# Inside the script, you can read and write to any of your
-# repositories directly:
-#
-#     App.Repo.insert!(%App.SomeSchema{})
-#
-# We recommend using the bang functions (`insert!`, `update!`
-# and so on) as they will fail if something goes wrong.
+App.Ctx.create_status(%{text: "verified"})
+App.Ctx.create_status(%{text: "unverified"})
```
Comment (nelsonic) [inline priv/repo/seeds.exs]: @SimonLab unverified should be `0` therefore we might need to insert it first ... 💭

File: priv/repo/seeds.exs [hunk 2]
```diff
@@ -1,11 +1,2 @@
...
-#     mix run priv/repo/seeds.exs
-#
-# Inside the script, you can read and write to any of your
-# repositories directly:
-#
-#     App.Repo.insert!(%App.SomeSchema{})
-#
-# We recommend using the bang functions (`insert!`, `update!`
-# and so on) as they will fail if something goes wrong.
+App.Ctx.create_status(%{text: "verified"})
```
Comment (nelsonic) [inline priv/repo/seeds.exs]: 👍

**Comment by SimonLab** [comment]
Test failing as the status seed value are not created when the test are running.
We can add a setup function (as we do for creating a loggedin person) but I'm wondering if there is a way to automatically run the seed also on before the tests

**Comment by nelsonic** [comment]
@SimonLab what about running the Status creation code in a _migration_ so it's only done once?

**Comment by SimonLab** [comment]
I've added the seed in the test alias:
```elixir
  defp aliases do
    [
      "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
      "ecto.reset": ["ecto.drop", "ecto.setup"],
      test: ["ecto.create --quiet", "ecto.migrate", "run priv/repo/seeds.exs", "test"]
    ]
  end
```
However I have some duplicate "verified" status which break the tests. I'm wondering if the tests data are not resetting at the end or if some tests are adding the value twice

**Comment by SimonLab** [comment]
My local database for tests is not reset, when it's done the tests are passing
`MIX_ENV=test mix ecto.drop` then `mix test` :heavy_check_mark:

**Pull request merged**